### PR TITLE
ACN_CD_CiscoUCS_Parser01

### PR DESCRIPTION
**Pull request by chicduong**
## Proposed Changes

  - Cisco UCS Parser Function
  -
  -

### CODE REVIEW

File: Parsers/CiscoUCS/CiscoUCS.txt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// Using functions in Azure monitor log queries: https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// 
+// LOG SAMPLES:
+// This parser assumes the raw log are formatted as follows:
+//
+// 1603399381.121    346 ::1 TCP_MISS/301 419 GET http://httpvshttps.com/ - HIER_DIRECT/45.33.7.16 text/html
+// 
+// 
+let LogHeader = Syslog
+| where Computer in ("UCSserver1", "UCSserver2")
```
Comment (shainw) [inline Parsers/CiscoUCS/CiscoUCS.txt:28]: This should have a comment here that these names are examples and must be changed to your server names.

File: Parsers/CiscoUCS/CiscoUCS.txt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+         Mneumonic = tostring(Parser[3]),
+         Substring = tostring(Parser[4])
+| project-away Parser;
+// Parse and normalize AUDIT logs
+let AuditLog = LogHeader
+| where Mneumonic == "AUDIT"
+| extend Parser = extract_all(@"^\[([\w\d\.\_\-]+)?\]\[([\w\d\.\_\-\\]+)?\]\[(\w+)?\]\[([\w+\_\-\.]+)?\]\[?(\w+)?\]?\[?([\w\d\.\_\-\\\/]+)?\]?\[?([\w\d\.\_\-\\\/]+)?\]?\[?([\w\d\.\_\-\\\/]+)?\]?([\S\s]+)?", dynamic([1,2,3,4,5,6,7,8,9]), Substring)
+| mv-expand Parser 
+| extend SrcUserName = tostring(Parser[0]),
+         DstUserName= tostring(Parser[1]),
```
Comment (shainw) [inline Parsers/CiscoUCS/CiscoUCS.txt:43]: structure consistency, need a space after DstUserName and '='

**Review by shainw — DISMISSED** [review]
I cannot test this as I don't have the data set, so I will assume it has been tested properly.  Couple of non-blocking comments that I would suggest are done, but not required.

**Comment by v-jayakal** [comment]
@chicduong, Please complete this PR, once this is merged into master, redirection URL can be generated required for https://github.com/Azure/Azure-Sentinel/pull/1281.

File: Parsers/CiscoUCS/CiscoUCS.txt (hunk 1 above)
Comment (chicduong) [reply Parsers/CiscoUCS/CiscoUCS.txt]: comments have been added

File: Parsers/CiscoUCS/CiscoUCS.txt (hunk 2 above)
Comment (chicduong) [reply Parsers/CiscoUCS/CiscoUCS.txt]: correction has been made

**Review by nazang — APPROVED** [review]

**Pull request merged**